CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ BUG FIXES:
 
  * cli: Fixed a bug where `nomad monitor -node-id` would cause a cli panic when no nodes where found. [[GH-6828](https://github.com/hashicorp/nomad/issues/6828)]
  * consul/connect: Fixed a bug where Connect-enabled jobs failed to validate when service names used interpolation. [[GH-6855](https://github.com/hashicorp/nomad/issues/6855)]
+ * agent: Fixed race condition in logging when using `nomad monitor` command [[GH-6872](https://github.com/hashicorp/nomad/issues/6872)]
```

Comment:
We tend to put them in alphabetical order, so I think this should go first in the list

Reply:
Wow! Can't believe I never realized that :sweat_smile: updated